continuous-integration/linux/install-sdist.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=$(realpath "$SCRIPT_DIR/../..")
+
+# Install source distribution
+pip install --upgrade pip
+pip install $ROOT_DIR/dist/*.tar.gz || exit $?
```

Comment:
Remember to quote all path arguments in case there is a whitespace in it

Reply:
Will do. I have to end the quote before the asterisk (*) though. If not it doesn't work (doesn't get expanded).